connector_importer/models/record.py

```diff
@@ -84,12 +92,12 @@ def debug_mode(self):
 
     @api.multi
     @job
-    def import_record(self, component_name, model_name):
+    def import_record(self, component_name, model_name, is_last_importer=True):
```

Comment:
I think it's worth leaving the small note on what does parameter `id_last_importer` does. As don't find it self-explanatory

Reply:
done